src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -56,6 +56,8 @@ public TypeWithAnnotations TypeWithAnnotations
             }
         }
 
+        public abstract FlowAnalysisAnnotations FlowAnalysisAnnotations { get; }
```

Comment:
This name was pretty confusing for me at first. It doesn't really say anything about nullable or that it's only about the attributes, not `?`

Reply:
This only represents the annotations from user-specified attributes. I agree this name is a bit confusing (we have annotations like `?` and flow analysis annotations like `[DisallowNull]`). I'll think about this for a subsequent PR.
Thanks